phpunit.xml.dist

```diff
@@ -0,0 +1,26 @@
+<phpunit
+	bootstrap="tests/bootstrap.php"
+	backupGlobals="false"
+	colors="true"
+	convertErrorsToExceptions="true"
+	convertNoticesToExceptions="true"
+	convertWarningsToExceptions="true"
+	>
+	<testsuites>
+		<testsuite name="default">
+			<directory suffix=".php">./tests/</directory>
+			<exclude>tests/test-edit-flow-custom-status-ajax.php</exclude>
+		</testsuite>
+		<!-- Sets the DOING_AUTOSAVE constant, so needs to be run last -->
+		<testsuite name="autosave">
```

Comment:
Rather than relying on order which is pretty fragile, it's better to use https://phpunit.de/manual/6.5/en/appendixes.annotations.html#appendixes.annotations.runTestsInSeparateProcesses to avoid leaking state between suites.

Reply:
Makes sense, I was just stealing from [wordpress-dev](https://github.com/WordPress/wordpress-develop/blob/master/phpunit.xml.dist#L13).

It's the `require_once` that's setting the `DOING_AJAX`, moving it into [setUp](https://github.com/Automattic/Edit-Flow/pull/546/commits/365511b5c30c83dd907cc454e15b570f0640874c#diff-df02cad6e7abc4e31f5cab9e94323a24R11) and adding the `runTestsInSeparateProcesses` seems to work